compiler/il/OMRBlock.hpp

```diff
@@ -233,7 +233,8 @@ class OMR_EXTENSIBLE Block : public TR::CFGNode
       CanCatchUserThrows          = 0x00000200,
       CanCatchOSR                 = 0x00000400,
       CanCatchOverflowCheck       = 0x00000800,
-      CanCatchEverything          = 0x000007FF, // Mask for all of the above
```

Comment:
The original value of `CanCatchEverything` is `0x000007FF` when the last `CanCatchOverflowCheck` is `0x00000800`. I think `CanCatchEverything` should've be `0x00000FFF`. Since I'm adding one more `CanCatchNewvalue` (`0x00001000`) now, `CanCatchEverything` is bumped to `0x00001FFF`